components/automate-ui/src/app/page-components/navbar/navbar.component.html

```diff
@@ -7,7 +7,9 @@
       <div class="navigation-menu">
         <div role="menuitem" class="nav-link"><a routerLink="/dashboards/event-feed" routerLinkActive="active" tabindex="0">Dashboards</a></div>
 
-        <div role="menuitem" class="nav-link"><a routerLink="/applications/service-groups" routerLinkActive="active" tabindex="0">Applications</a></div>
+        <app-authorized [anyOf]="['/auth/introspect', 'get']">
```

Comment:
```suggestion
        <app-authorized [anyOf]="['/applications', 'get']">
```

pretty sure thats what you want there..

Reply:
@susanev is correct; you will want this one... at least. See my notes below.

Reply:
also, the reason this appears to work is because your policy has `applications:*` in it. i imagine though that that page would have errors if the user (by way of the policy) only has access to services and not the service groups.